src/Toolbox.php

```diff
@@ -3463,4 +3464,25 @@ public static function cleanPaths(string $message): string
     {
         return ErrorUtils::cleanPaths($message);
     }
+
+    public static function iniSet(
+        string $name,
+        string $value,
+        string $loglvl = LogLevel::WARNING
+    ): void
+    {
+        try {
+            ini_set($name, $value);
+        } catch (InfoException $e) {
+            self::log(
+                $loglvl,
+                sprintf(
+                    'Unable to set `%s` to `%s`. You should enforce the value in your PHP configuration. Error is: %s',
```

Comment:
In #21236, this message was added as the corresponding ini entries are expected to be always configured the same for the whole GLPI context.

For the current PR, it is different. We do not want people to enforce `memory_limit` to `1024M` or `max_execution_time` to `300`.

```suggestion
                    'Unable to set `%s` to `%s`. This may be caused by a `php_admin_flag` or a `php_admin_value` directive in your web server configuration. Try to use `php_flag` or `php_value` instead. Error is: %s',
```